accounts/abi/unpack_test.go

```diff
@@ -724,7 +724,8 @@ func TestUnmarshal(t *testing.T) {
 	buff := new(bytes.Buffer)
 
 	// marshall mixed bytes (mixedBytes)
-	p0, p0Exp := []byte{}, common.Hex2Bytes("01020000000000000000")
+	var p0 []byte
+	p0Exp := common.Hex2Bytes("01020000000000000000")
```

Comment:
This didn't really help code readability (it was readable before), and you didn't change p1 to a similar declaration to keep things consistent. I think it's fine to keep this code as is. It's clean enough imho.

Reply:
Done.